templates/Entry/entry.html.twig

```diff
@@ -320,7 +320,7 @@
             </div>
 
             {% if is_granted('TAG', entry) %}
-                <div class="input-field nav-panel-add-tag" style="display: none">
+                <div class="input-field nav-panel-add-tag hidden" data-controller="add-tag" data-action="leftbar:toggleAddTagForm@window->add-tag#toggle">
```

Comment:
same trick here about dispatching an event and listening to it on window from another controller.
it's a way to interact between separated controllers